client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -181,11 +181,42 @@ class SimplePaymentsDialog extends Component {
 		return isNumber( this.props.editPaymentId );
 	}
 
+	checkUnsavedForm() {
+		const formIsUnsaved = this.state.activeTab === 'form' && this.props.formIsDirty;
```

Comment:
Although the state is defined in the constructor maybe we could get the `activeTab` using `get()`.

```es6
const formIsUnsaved = get( this.state, 'activeTab' ) === 'form' && this.props.formIsDirty;
```